src/map/script.c

```
@@ -12865,42 +12901,40 @@ static BUILDIN(getwaitingroomstate)
 		script_pushint(st, -1);
 		return true;
 	}
 
-	switch(type) {
-		case 0:
-			for (i = 0; i < cd->users; i++) {
-				struct map_session_data *sd = cd->usersd[i];
-				nullpo_retr(false, sd);
-				mapreg->setreg(reference_uid(script->add_variable("$@chatmembers"), i), sd->bl.id);
-			}
-			script_pushint(st, cd->users);
-			break;
-		case 1:  script_pushint(st, cd->limit); break;
-		case 2:  script_pushint(st, cd->trigger&0x7f); break;
-		case 3:  script_pushint(st, ((cd->trigger&0x80)!=0)); break;
-		case 4:  script_pushstrcopy(st, cd->title); break;
-		case 5:  script_pushstrcopy(st, cd->pass); break;
-		case 16: script_pushstrcopy(st, cd->npc_event);break;
-		case 32: script_pushint(st, (cd->users >= cd->limit)); break;
-		case 33: script_pushint(st, (cd->users >= cd->trigger)); break;
-
-		case 34: script_pushint(st, cd->min_level); break;
-		case 35: script_pushint(st, cd->max_level); break;
-		case 36: script_pushint(st, cd->zeny); break;
-		default: script_pushint(st, -1); break;
+	switch (type) {
+	case 0:
+		for (i = 0; i < cd->users; i++) {
+			struct map_session_data *sd = cd->usersd[i];
+			nullpo_retr(false, sd);
+			mapreg->setreg(reference_uid(script->add_variable("$@chatmembers"), i), sd->bl.id);
+		}
+		script_pushint(st, cd->users);
+		break;
+	case 1:  script_pushint(st, cd->limit); break;
+	case 2:  script_pushint(st, cd->trigger & 0x7f); break;
+	case 3:  script_pushint(st, ((cd->trigger & 0x80) != 0)); break;
+	case 4:  script_pushstrcopy(st, cd->title); break;
+	case 5:  script_pushstrcopy(st, cd->pass); break;
+	case 16: script_pushstrcopy(st, cd->npc_event); break;
+	case 32: script_pushint(st, (cd->users >= cd->limit)); break;
+	case 33: script_pushint(st, (cd->users >= cd->trigger)); break;
+	case 34: script_pushint(st, cd->min_level); break;
+	case 35: script_pushint(st, cd->max_level); break;
+	case 36: script_pushint(st, cd->zeny); break;
+	default: script_pushint(st, -1); break;
```

should throw warning invalid type and return false;

---

added